doc/API.md

```diff
@@ -527,6 +535,23 @@ axe.run(document, {
   console.log(results);
 });
 ```
+
+#### Example 4
+
+This example shows a result object that points to a shadow DOM element.
+
+* `violations[0]`
+  * `help` - `"Elements must have sufficient color contrast"`
+  * `helpUrl` - `"https://dequeuniversity.com/rules/axe/2.1/color-contrast?application=axeAPI"`
+  * `id` - `"color-contrast"`
+		* `nodes`
+			* `target[0][0]` - `"header > aria-menu"`
+			* `target[0][1]` - `"li.expanded"`
+
+* `violations[1]` ...
+
+As you can see the `target` array contains one item that is an array. This array contains two items, the first is a CSS selector string that finds the custom element `<aria-menu>` in the `<header>`. The second item in this array is the selector within that custom element's shadow DOM to find the `<li>` element with a class of `expanded`.
```

Comment:
> As you can see the `target` array contains one item that is an array.

I see two items...? Perhaps this should be spelled out more completely since I wasn't able to "see" it?